Gemfile

```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
-ruby '2.3.0'
 
```

Comment:
This is needed for RVM (or rbenv) to know what version of Ruby the repo uses, so that all developers are using the same Ruby version. We can either do that via `.ruby-version` or by specifying it in the Gemfile. Is there a particular reason we need to run two patch versions in Travis? Can we make it simple and only support the latest at any given time?